src/components/Fields/TextArea.js

```diff
@@ -23,6 +23,7 @@ const propTypes = {
     isValid: PropTypes.bool,
     label: PropTypes.string,
     name: PropTypes.string,
+    autoComplete: PropTypes.oneOf(['on', 'off']),
```

Comment:
All props in ui-react should be ordered alphabetically. Please amend here, in the defaultProps definition and in the JSX.